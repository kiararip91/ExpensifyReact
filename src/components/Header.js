import React from 'react'
import {NavLink} from 'react-router-dom'

const Header = () => (
    <div>
        <header>Expensify</header>
        <NavLink to='/' activeClassName="is-active" exact={true}>Home</NavLink>
        <NavLink to='/create' activeClassName="is-active">New Expense</NavLink>
        <NavLink to='/help' activeClassName="is-active">Help</NavLink>
    </div>
)

export default Header