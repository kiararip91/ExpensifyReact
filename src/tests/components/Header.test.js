import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import React from 'react'
import Header from '../../components/Header'

test('Should render header correctly', () => {
    const wrapper = shallow(<Header />)
   // expect(wrapper.find('h1').text()).toBe("Expensify")
   expect(toJSON(wrapper)).toMatchSnapshot()
})