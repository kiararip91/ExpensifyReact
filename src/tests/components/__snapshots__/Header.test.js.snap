// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render header correctly 1`] = `
<div>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    New Expense
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/help"
  >
    Help
  </NavLink>
</div>
`;
